db/migrate/20191025141802_create_application_experience.rb

```diff
@@ -0,0 +1,22 @@
+class CreateApplicationExperience < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_experiences do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :title, null: false
```

Comment:
`title` feels too ambiguous here. 
I think that `role` better describes this field.